docs/Reference/Dropdown.md

```diff
@@ -851,10 +851,10 @@ var dropdown = new kintoneUIComponent.Dropdown({
     ],
     value: 'Banana'
 });
- 
+
 var body = document.getElementsByTagName("BODY")[0];
-    body.appendChild(dropdown.render();
-    
+    body.appendChild(dropdown.render()));
```

Comment:
@r3-yonashiro -san
The ')' before ';' is redundant, please remove it for using.

Reply:
fixed! Thanks!